packages/ui/components/ui/tx/view/address-view.tsx

```diff
@@ -47,7 +49,7 @@ export const AddressViewComponent = ({
         <CopyToClipboard
           text={encoded}
           label={
-            <div>
+            <div data-testid='AddressView__CopyIcon'>
```

Comment:
Do we have strong feelings one way or the other about test IDs? I avoid them whenever possible, but couldn't think of a better way to test that the copy icon is present.

Reply:
I don't think it's a big deal, but in theory you can just query the container right?

```typescript
const { container } = render(<AddressViewComponent />);
const svgSelector = 'svg.h-4.w-4.text-muted-foreground.hover\\:opacity-50';
const divWithSvg = container.querySelector(`div > ${svgSelector}`);
expect(divWithSvg).toBeDefined() // or something like this
```

Reply:
Hm... that'd be pretty brittle though. If we end up deciding to go with `h-8` in the future, these tests would break for reasons unrelated to the test's purpose. If it's no big deal to you either way, I'll leave it as-is.